nvdaHelper/espeak/sconscript

```diff
@@ -426,10 +446,14 @@ for i in phonemeData:
 	fileName = i.abspath[l:]
 	env.InstallAs(os.path.join(synthDriversDir.Dir('espeak-ng-data').abspath, fileName), i)
 
+
+# Removes files that are created when installing from dictsource/extra/*_* to dictsource.
+# Also removes emoji files from dictsource compilation, refer to cleanEmoji for justification.
+env.AddPreAction(espeakLib, env.Action(cleanFiles_preBuildAction))
 # Move any extra dictionaries into dictsource for compilation
 env.Install(
 	espeakRepo.Dir('dictsource'),
-	env.Glob(os.path.join(espeakRepo.abspath, 'dictsource', 'extra', '*'))
+	env.Glob(os.path.join(espeakRepo.abspath, 'dictsource', 'extra', '*_*'))

```

Comment:
What is this change for?

Reply:
This undo's a change from a previous commit. These are functionally the same currently. The eSpeak build script uses a wider glob `'*'` in a similar check, however I thought it might be better to keep it as-is.